stats/datadog/collector_test.go

```diff
@@ -4,19 +4,18 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/loadimpact/k6/lib"
 	"github.com/loadimpact/k6/stats"
 	"github.com/loadimpact/k6/stats/statsd/common"
 	"github.com/loadimpact/k6/stats/statsd/common/testutil"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCollector(t *testing.T) {
-	var tagSet = lib.GetTagSet("tag1", "tag2")
-	var handler = tagHandler(tagSet)
+	var tagMap = stats.TagSet{stats.TagProto.String(): true, stats.TagGroup.String(): true}
```

Comment:
Why was the previous test modified, what was wrong with `tag1` and  `tag2`?

Reply:
https://github.com/loadimpact/k6/pull/1148#discussion_r334887048